modules/internal/ChapelGpuSupport.chpl

```diff
@@ -61,8 +61,10 @@ module ChapelGpuSupport {
   }
 
   private proc isEnvSet(name: string): bool {
-    extern proc getenv(name : c_string) : c_string;
-    var env = bytes.createBorrowingBuffer(getenv(name.localize().c_str()));
+    private use CTypes;
+    extern proc getenv(name : c_ptrConst(c_char)) : c_ptrConst(c_char);
+    const localName = name.localize();
+    const env = bytes.createBorrowingBuffer(getenv(localName.c_str()));
```

Comment:
Let's keep it as `name.localize().c_str()` here

Reply:
oops, thought I fixed all these earlier, thanks!